extensions/amp-bind/0.1/test/test-bind-integration.js

```diff
@@ -66,9 +69,10 @@ describe.configure().retryOnSaucelabs().run('integration amp-bind', function() {
 
     it('should update CSS class when class binding changes', () => {
       const textElement = fixture.doc.getElementById('textElement');
-      const button = fixture.doc.getElementById('mutateTextButton');
+      const changeClassButton =
```

Comment:
Nit: When naming, be only as specific as necessary to disambiguate. I.e. in the absence of other buttons in this scope, just `button` would suffice. Otherwise readers have to take some time to understand what the "changeClass" prefix signifies.